backend/README.md

```diff
@@ -0,0 +1,42 @@
+# How to run locally
+
+## JAVA
+
+You need java 11 to run the application.
+
+##MAVEN
+There is a maven executable for unix-base dos in the backend repo called mvnw. To execute it, run `./mvnw <comands>`
+
+- Maven is a dependency manager for java project. Using the cli commands, you can import dependency, build project, run test . . .
+
+- To import all dependency and create the jar files and run the tests, run `./mvnw clean install`
+
+- To import all dependency and create jar file without running tests, run `./mvnw/ clean install -DskipTests`
+
+- To run only the tests run `./mvnw clean test`
+
+## Neo4j
+You need to install neo4j in your OS of choice. For macOS users, you can use brew by running `brew install neo4j` in your terminal.
+
+- You can use a GUI to look at your local neo4j instance using neo4j desktop (https://neo4j.com/download-neo4j-now/).
+
+- When you first start neo4j, the default username and password is neo4j (for both). neo4j will not allow any application
+to connect if the password has not been changed.
+
+- To start neo4j locally, open your terminal and run `neo4j start`
+
+- If neo4j is up and running locally, you can run `curl -v -u neo4j:neo4j -X POST localhost:7474/user/neo4j/password -H "Content-type:application/json" -d "{\"password\":\"secret\"}"` and replace the secret string with stonecap. stonecap is our db password for dev when running the application locally.
```

Comment:
I would add the password directly into the code like 
`curl -v -u neo4j:neo4j -X POST localhost:7474/user/neo4j/password -H "Content-type:application/json" -d "{\"password\":\"stonecap\"}"`

so people can easily copy paste it. You can also mention that if this doesn't work, to go through the browser at 'localhost:7474'